### ISSUE TITLE
bridges run perpendicular to the desired direction

### ISSUE THREAD

**Issue by SurfBlaster** [rvmn/Pleccer#15]
All bridges run perpendicular to the desired direction - that is, along the long edges of the pit.

Maybe this can be fixed with some slicer settings, but I did not find such settings.

ver. alpha5_prerelease
![bridge-sp-wtf](https://github.com/rvmn/SuperPleccer/assets/69464650/cc1a4497-dc81-4c58-a78b-b8a6a09ea56a)

**Comment by pleccer** [comment]
You can change the general setting of all the bridges using the bridge angle setting under infill. 
in the current dev state i set it so it will choose the direction that causes shortest bridging, i will push it in a few days

**Comment by SurfBlaster** [comment]
Thanks, this really helped as a temporary solution!

infill - advanced - angle - bridging = 45

Offtopic:

However, printing my part is still not possible. Most arc-supports start in the air. And with a change in this angle, there are even more of them. And besides, an unexpected effect, changing this angle has caused the center of the support circles to now shift from the center of the polygon to its corner.

before/after

![sp-arc-more-or-less-1](https://github.com/rvmn/SuperPleccer/assets/69464650/f999b556-b08d-4559-b8c4-c39c7a1eb3dc)
![sp-arc-more-or-less-2](https://github.com/rvmn/SuperPleccer/assets/69464650/dc525a4a-fae8-459c-81f8-a7dce5a1648c)

**Comment by SurfBlaster** [comment]
> i will push it in a few days

My apologies, is there any progress?

**Comment by rvmn** [comment]
yes, it is taking a bit longer than i was planning, apologies for that. 
i am spending the last hand on this tomorrow and the next days, ETA is before the end of the week.
thanks for the bump

**Comment by j-snijder** [comment]
Hi, any idea on what timetable for the new windows compiled version ?Doing a great jib, thanks for all your effort SP beats PS by miles 😁Op 10 jul. 2023 10:13 schreef rvmn ***@***.***>:
yes, it is taking a bit longer than i was planning, apologies for that.
i am spending the last hand on this tomorrow and the next days, ETA is before the end of the week.
thanks for the bump
